.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.3
```

Comment:
Any reason why 2.5.3, when base images are only available for 2.6 & 2.7 https://github.com/simplybusiness/brave_new_world/blob/master/docs/BASE_IMAGES.md?

Reply:
This gem is open sourced and been here for a while, so I thought it's better to use some older version. But I don't have a strong preference tbh.